#!/usr/bin/env python
PACKAGE = "rm_tracking_pid"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("y_kp",    double_t,    0, "Kp for y axis", 0.0,  0, 5)
gen.add("y_kd",    double_t, 0, "Kd for y axis", 0.0, 0,   5)

gen.add("z_kp",    double_t,    0, "Kp for z axis", 0.0,  0, 5)
gen.add("z_kd",    double_t, 0, "Kd for z axis", 0.0, 0,   5)

exit(gen.generate(PACKAGE, "rm_tracking_pid", "tuning"))
